### PR TITLE
dbeaver/pro#2925 Add schema and catalog to qm

### PR DESCRIPTION
**Pull request by HocKu7**
Added schema and catalog to view of the record in qm
image

### CODE REVIEW

**Comment by E1izabeth** [comment]
Please provide a screenshot

File: plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/controls/querylog/QueryLogViewer.java [hunk 1]
```diff
@@ -1082,8 +1082,16 @@ protected Composite createDialogArea(Composite parent) {
             messageLabel.setLayoutData(new GridData(GridData.VERTICAL_ALIGN_BEGINNING));
 
             Control msg;
-            if (object.getObject() instanceof QMMStatementExecuteInfo) {
+            if (object.getObject() instanceof QMMStatementExecuteInfo qmmStatementExecuteInfo) {
                 msg = createSQLPanel(topFrame);
+                Composite sqlDetailsPanel = UIUtils.createPlaceholder(topFrame, 4, 5);
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/controls/querylog/QueryLogViewer.java:1087]: I don't like that labels are misaligned. Let's wrap catalog and schema names so they are each on a separate line.

**Comment by Matvey16** [comment]
Verified

File: plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/controls/querylog/QueryLogViewer.java (hunk 1 above)
Comment (serge-rider) [reply plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/controls/querylog/QueryLogViewer.java:1087]: Use createComposite()

**Review by serge-rider — CHANGES_REQUESTED** [review]

**Pull request merged**